src/semantic/error.rs

```diff
@@ -33,6 +33,17 @@ enum SemanticErrorVariant {
     OnlyTraitCanBeSuperType { got_type: Type },
     #[error("dead code after return statement")]
     DeadCodeAfterReturnStatement,
+    #[error("redefined field `{}` in class `{}`, already defined at {}", .field_name, .class_name, .previous_definition)]
```

Comment:
@YanHerChen Seems like I forget to change naming, it should be `member`, not `field`. Please do request changes here